pkgs/tools/package-management/nix/2_26/packaging/components.nix

```diff
@@ -20,13 +29,279 @@ let
       }_${src.shortRev or "dirty"}";
 
   fineVersion = baseVersion + fineVersionSuffix;
+
+  root = ../.;
+
+  # Indirection for Nixpkgs to override when package.nix files are vendored
+  filesetToSource = lib.fileset.toSource;
```

Comment:
Was our regression fix for fileset also backported to older version? People from time to time have outdated installer with old nix versions, so it would be annoying if we would break nixos-install for them.